flexget/plugins/operate/free_space.py

```diff
@@ -22,100 +62,66 @@ def get_free_space(config, task):
         ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
         try:
             ssh.connect(
-                config.get('host'),
-                config.get('port', 22),
-                config.get('user'),
-                config.get('password', None),
-                config.get('pkey', None),
```

Comment:
Password and pkey were never in the schema. Unstructured config didn't really let us see that easily.